### PR TITLE
A few simple editorial fixes from the ART-ART review.

### LINKED ISSUE #899: ART-ART: Remove first sentence of API summary
As you can tell, I don't think the documents do what the first sentence of the
API summary claims. This document would not lose anything if you deleted the
sentence.

### PR DESCRIPTION
**Pull request by mwelzl**
Resolves #899, #902, #903, #904, #906, #907, #916.

### CODE REVIEW

File: draft-ietf-taps-interface.md [hunk 1]
```diff
@@ -299,10 +299,6 @@ provides:
 
 # API Summary
 
-The Transport Services API is the basic common abstract application
-programming interface to the Transport Services Architecture defined in the TAPS
-Architecture {{I-D.ietf-taps-arch}}.
-
```
Comment (gorryfair) [inline draft-ietf-taps-interface.md:305]: Why remove this - isn't the API the inerface spec for the architecture?

Comment (mwelzl) [reply draft-ietf-taps-interface.md:305]: See #899

Comment (gorryfair) [reply draft-ietf-taps-interface.md:305]: OK, understood -see issue.

File: draft-ietf-taps-interface.md [hunk 2]
```diff
@@ -1830,11 +1831,13 @@ Default:
...
 
 This property specifies the minimum number of bytes in a received
-message that need to be covered by a checksum. A special value of 0 means
-that a received packet does not need to have a non-zero checksum field. 
+message that need to be covered by a checksum. 
 A receiving endpoint will not forward messages that have less coverage 
 to the application. The application is responsible for handling
 any corruption within the non-protected part of the message {{!RFC8085}}.
+A special value of 0 means
+that a received packet may also have a zero checksum field.
```
Comment (tfpauly) [inline draft-ietf-taps-interface.md:1839]:
```suggestion
A special value of 0 means that a received packet may also have a zero checksum field.
```

**Review by tfpauly — APPROVED** [review]

**Pull request merged**